data/config_files.yaml

```diff
@@ -21,8 +21,8 @@ sources:
 - type: FILE
   attributes:
     paths:
-    - '%%environ_programfiles%%\Redis\conf\redis.windows.conf'
-    - '%%environ_programfiles%%\Redis\conf\redis.conf'
+    - '%%environ_programfiles%%/Redis/conf/redis.windows.conf'
+    - '%%environ_programfiles%%/Redis/conf/redis.conf'
```

Comment:
please use `\` and add `separator: \` to keep it closest to the OS path as possible